hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/ActionRepository.java

```diff
@@ -225,6 +225,17 @@ void switchStatus(@Param("statusToSet") Action.Status statusToSet, @Param("targe
     List<Long> findByTargetIdInAndIsActiveAndActionStatusAndDistributionSetNotRequiredMigrationStep(
             @Param("targetsIds") List<Long> targetIds, @Param("active") boolean active,
             @Param("currentStatus") Action.Status currentStatus);
+    
+    /** Retrieves all {@link Action}s that matches the queried externalRefs.
+     * @param externalRefs
+     *            for which the actions need to be found
+     * @param active
+     *            flag to indicate active/inactive actions
+     * @return list of actions 
+     */
+    @Query("SELECT a FROM JpaAction a WHERE a.externalRef IN :externalRefs AND a.active = :active")
```

Comment:
do we still need to write this query ourselves? wouldn't the method name be enough for Jpa?

Reply:
Yes you are right. We dont need the query.